src/fibonacci.js

```diff
@@ -6,6 +6,26 @@
  * @returns {number[]}  - Fibonacci Serie
 */
 
-function fibonacci(limit) { }
+function fibonacci(limit) {
+  if (limit < -1) {
+    throw new Error(`Invalid Input Negative Number: ${n}`);
+  }
+  const arr = [];
+  arr.push(0);
+  if (limit === 0) {
```

Comment:
This is the same as ```ts if(!limit) { ... }```. Zero is the only falsy number. Any other number is truthy.

Reply:
But `limit === 0` is OK for double checking given that JS doesn't ensure data types.